requirements.txt

```diff
@@ -1,2 +1,2 @@
 requests>=2.22.0
-pydantic==1.*
```

Comment:
Allows, the end user of the sdk to pin <2 on their side. This just allows us to unblock people who do want to move to pydantic v2+